src/main/java/mate/controller/car/DeleteCarController.java

```diff
@@ -16,6 +18,6 @@
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws IOException {
         carService.delete(Long.parseLong(req.getParameter("id")));
-        resp.sendRedirect("/cars/all");
+        resp.sendRedirect("/cars/");
```

Comment:
я не знайшла контроллер з патерном /cars/
```suggestion
        resp.sendRedirect(""/cars/all");
```